crates/dojo-world/src/migration/mod.rs

```diff
@@ -205,19 +203,19 @@ pub trait Deployable: Declarable + Sync {
         let InvokeTransactionResult { transaction_hash } =
             txn.send().await.map_err(MigrationError::Migrator)?;
 
-        // TODO: remove finality check once we can remove displaying the block number in the
-        // migration logs
-        let receipt = TransactionWaiter::new(transaction_hash, account.provider())
-            .with_finality(TransactionFinalityStatus::AcceptedOnL2)
-            .await
-            .map_err(MigrationError::WaitingError)?;
+        // // TODO: remove finality check once we can remove displaying the block number in the
+        // // migration logs
+        // let receipt = TransactionWaiter::new(transaction_hash, account.provider())
+        //     .with_finality(TransactionFinalityStatus::AcceptedOnL2)
+        //     .await
+        //     .map_err(MigrationError::WaitingError)?;
```

Comment:
Hm are we removing this?

Reply:
Aah, yeah. We had a chat about this in the team meet. So that migration is faster. We skip the transaction waiter for block number. I commented it so we can bring it back if we need.

Reply:
i believe u might've misunderstood.

it's slow bcs the `TransactionWaiter` will wait for the tx to be `ACCEPTED_ON_L2` which mean it has to wait until the tx is no longer in the `pending` block. Because we need to get the block number attached to the the tx receipt, which is only available in `non-pending` tx receipt

Reply:
Replaced this and some others with `TransactionWaiter::new(transaction_hash, account.provider()).await?;`
Does that seem all right?